### PR TITLE
HW_Lesson_14 

### PR DESCRIPTION
**Pull request by Jollion81**
This is Pull Request for HW 14 lesson.

### CODE REVIEW

**Comment by landriiuk** [comment]
@Jollion81 hi, add pls structure to your code, as we discussed till tomorrow

**Comment by Jollion81** [comment]
> @Jollion81 hi, add pls structure to your code, as we discussed till tomorrow

Accepted.

File: cypress.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (landriiuk) [inline cypress.json]: here should be added baseUrl as minimum

File: package.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "cypress_automation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (landriiuk) [inline package.json]: Add command here how run your tests

File: cypress/integration/lesson14_spec.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+describe('My First Test', () => {
+  before(() => {
+    cy.visit('https://demoqa.com')
```
Comment (landriiuk) [inline cypress/integration/lesson14_spec.js]: After added base url into cypress.json u will use 
```suggestion
    cy.visit('/')
```

File: cypress/integration/lesson14_spec.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    })
+
+    after(() => {
+      cy.reload(); // cleanup text from the inputs fields
```
Comment (landriiuk) [inline cypress/integration/lesson14_spec.js]: better it add to before each, due to cypress bug

File: cypress/integration/lesson14_spec.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    });
+      // Should be on a new URL which includes '/elements'
```
Comment (landriiuk) [inline cypress/integration/lesson14_spec.js:10]: Delete this comments. U have test name, it's enough
Your test name should be more meaningfull

**Review by landriiuk — APPROVED** [review]

**Comment by landriiuk** [comment]
Fix comments^ after that u are able to merge into master

Score 8/10

File: cypress.json (hunk 1 above)
Comment (Jollion81) [reply cypress.json]: Accepted.

File: package.json (hunk 2 above)
Comment (Jollion81) [reply package.json]: Accepted.

File: cypress/integration/lesson14_spec.js (hunk 3 above)
Comment (Jollion81) [reply cypress/integration/lesson14_spec.js]: Accepted.

File: cypress/integration/lesson14_spec.js (hunk 4 above)
Comment (Jollion81) [reply cypress/integration/lesson14_spec.js]: Accepted.
Fixed.

File: cypress/integration/lesson14_spec.js (hunk 5 above)
Comment (Jollion81) [reply cypress/integration/lesson14_spec.js:10]: Accepted.

**Pull request merged**